hyperion-sample/build.gradle

```diff
@@ -35,6 +35,9 @@ dependencies {
     implementation "com.android.support:appcompat-v7:${project.versionSupportLibrary}"
     implementation "com.android.support:recyclerview-v7:${project.versionSupportLibrary}"
     implementation "com.jakewharton.timber:timber:4.7.0"
+    implementation "android.arch.persistence.room:runtime:1.1.0"
+    implementation 'com.android.support.constraint:constraint-layout:1.1.0'
```

Comment:
I may have missed something; why does the sample app need constraint layout?

Reply:
I believe constraint layout is added by default now so this would be the same issue as the auto-generated tests.